utils/google-document.js

```diff
@@ -122,7 +124,13 @@ class GoogleDocument {
     text = text.replace(/_/g, "\\_") // Prevent _ to be italic
 
     if (baselineOffset === "SUPERSCRIPT") {
-      text = `<sup>${text}</sup>`
+      if (this.enableFontSize) {
+        text = `<span style="font-size:${
+          fontSize.magnitude
```

Comment:
something like this would work?

Reply:
You must do not put this inside `if (baselineOffset === "SUPERSCRIPT") {` 
It's a different transformation